src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -1,5 +1,6 @@
 import errno
 import json
+import yaml
```

Comment:
We might need this package dependency in `ceph.spec.in`

Reply:
Do we really need this?  We're trying to limit ourselves to the base python install.  Can you use json instead of yaml?